src/App.js

```diff
@@ -15,20 +15,40 @@ chance.mixin({
 });
 
 const allFriends = Array.from({length: 20}, () => chance.friend());
+const bestFriends = allFriends.filter(friend => friend.isBestFriend);
 
 console.log(allFriends);
 
 export default class App extends Component {
   render() {
     return (
       <div className="App">
-
-        {/*Make a list of all your friends*/}
-
-        {/*Make a list of just your best friends*/}
-
-        {/*Show the average age of your best friends*/}
-
+          <div className="friendList">
+              {/*Make a list of all your friends*/}
+              <h1>My Friends</h1>
+              <ul>
+                  {allFriends.map((friend, index) =>
+                      <li key={index}>
+                          {friend.first} {friend.last}
+                      </li>
+                  )}
+              </ul>
+              {/*Make a list of just your best friends*/}
+              <h1>My Best Friends</h1>
+              <ul>
+                  {bestFriends.map((friend, index) =>
+                      <li key={index}>
+                          {`${friend.first} ${friend.last}`}
+                      </li>
+                  )}
+              </ul>
+              {/*Show the average age of your best friends*/}
+              <h1>Average Age of My Best Friends</h1>
+              <ul>
```

Comment:
Overall very neat code! The average age could just be in a paragraph block instead of in a list block.